packages/cli/lib/errorHandlers/overrrideErrors.js

```diff
@@ -0,0 +1,95 @@
+const { isSpecifiedError } = require('@hubspot/local-dev-lib/errors/apiErrors');
+const { logger } = require('@hubspot/local-dev-lib/logger');
+
+const { PLATFORM_VERSION_ERROR_TYPES } = require('../constants');
+const { i18n } = require('../lang');
+const { uiLink } = require('../ui');
+
+const i18nKey = 'cli.lib.errorHandlers.overrideErrors';
+
+function createDashes() {
+  return '-'.repeat(79);
+}
+
+function createPlatformVersionError(subCategory, errorData) {
+  const platformVersion = errorData.context.PLATFORM_VERSION[0];
+  const docsLink = uiLink(
+    i18n(`${i18nKey}.platformVersionErrors.docsLink`),
+    'https://developers.hubspot.com/docs/platform/platform-versioning'
+  );
+  let translationKey;
+
+  if (
+    subCategory === PLATFORM_VERSION_ERROR_TYPES.PLATFORM_VERSION_NOT_SPECIFIED
+  ) {
+    translationKey = 'unspecifiedPlatformVersion';
+  } else if (
+    subCategory === PLATFORM_VERSION_ERROR_TYPES.PLATFORM_VERSION_RETIRED
+  ) {
+    translationKey = 'platformVersionRetired';
+  } else if (
+    subCategory ===
+    PLATFORM_VERSION_ERROR_TYPES.PLATFORM_VERSION_SPECIFIED_DOES_NOT_EXIST
+  ) {
+    translationKey = 'nonExistentPlatformVersion';
+  }
+
+  logger.log(createDashes());
+  logger.log(i18n(`${i18nKey}.platformVersionErrors.header`));
```

Comment:
I broke up the text a bit to make the i18n translations more contained, especially since I needed to interpolate the platform version and docs link. Is there a better approach here?

Reply:
This looks good to me 👍. Should this top log line be using `logger.error()`? Then you wouldn't need to manually insert the "[ERROR]" [here](https://github.com/HubSpot/hubspot-cli/pull/1016/files#diff-f1b852efd5f578b3521c0020953a63d55a5670e3bfe42219ffb0f77bae7c4cdeR1346).

Reply:
Ooh nice, thanks Branden!